### ISSUE TITLE
Screen readers are not announcing state of "Vary by configuration" menu options in project property pages when focused

### ISSUE THREAD

**Issue by japarson** [dotnet/project-system#8003]
## Visual Studio Version
Microsoft Visual Studio Enterprise 2022 Preview
Version 17.2.0 Preview 1.0

## Summary
Screen readers are not announcing state of "Vary by configuration" menu options in project property pages when focused.
- [Bug 1504039](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1504039): A11y_Visual Studio Client_Android options_Code shrinker_Info and relationships: Narrator/NVDA is not announcing the selected/unselected state of the menu item

https://user-images.githubusercontent.com/59936622/159622851-4ede437b-0a13-46f2-b5e1-79d7bb710507.mp4

## Steps to Reproduce.
1. Turn on Narrator/NVDA.
1. Create new Android project from power shell using `dotnet new android -n <project_name>`.
1. Open VS 2022 and tab till "Open a project or solution" option and press enter.
1. Tab till "File name" and select newly created iOS "csproj" file and press enter on "Open" button.
1. Tab till project in "Solution explorer" and press context key.
1. Tab till "Properties" menu item and press enter.
1. Tab till "Android Options" button and press enter.
1. Tab till "Debug" field and select "APK" option.
1. Tab till "Code shrinker" section present in "Packaging".
1. Tab till menu item and press enter.
1. Check whether the screen reader is reading the selected/unselected state of menu item or not.

## Expected Behavior
Narrator/NVDA should announce the selected/unselected state of the menu item.

## Actual Behavior
Narrator/NVDA is not announcing the selected/unselected state of the menu item.

## User Impact
Screen reader users may not be able to know the information about which item is selected/unselected.

[MAS 1.3.1 Info and Relationships](https://liquid.microsoft.com/Web/Views/View/410630#Zrex-3A-2F-2Fms-2Eaccessibility-2FRequirements-2F01-2E03-2E01-2F)

**Comment by adamint** [comment]
PR is: https://devdiv.visualstudio.com/DevDiv/_git/CPS/pullrequest/394189

**Comment by japarson** [comment]
Is this fix now available in VS main since the PR is merged?

**Comment by adamint** [comment]
Yes @japarson
